programs/voter-stake-registry/src/state/deposit_entry.rs

```diff
@@ -400,4 +422,77 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    pub fn far_future_lockup_start_test() -> Result<()> {
+        let day: i64 = 86_400;
+        let saturation: i64 = 5 * day;
+        let deposit = DepositEntry {
+            amount_deposited_native: 10_000,
+            amount_initially_locked_native: 10_000,
+            lockup: Lockup {
+                start_ts: 10_000_000_000,
```

Comment:
add comment to what this represents e.g. Saturday, November 20, 2286 5:46:40 PM

Reply:
btw instead of arbitrary number shouldnt we choose one which is max of below integer type +/- something so that underflow/overflow are actually triggered

Reply:
I don't think the date represented by that number matters. The underflow triggered before because the seconds to the first vesting event were larger than the lockup saturation time. That's the case in this test.

Reply:
added comments